### PR TITLE
Add initial vendors rejection

### PR DESCRIPTION
**Pull request by osk-8**
Initial rejection occurs when:
- User visit vendors list view for the first time;
- User visit purpose tab for the first time;

### CODE REVIEW

File: src/components/popup/details/vendors/vendors.jsx [hunk 1]
```diff
@@ -52,7 +55,7 @@ export default class Vendors extends Component {
 
 	isFullVendorsConsentChosen = () => {
 		const {vendors, selectedVendorIds} = this.props;
-		return vendors.length === selectedVendorIds.size;
+		return !vendors.filter(({id}) => !selectedVendorIds.has(id)).length;
```
Comment (mvciekzvbek) [inline src/components/popup/details/vendors/vendors.jsx]:
```suggestion
		const isPresent = ({id}) => selectedVendorIds.has(id);
		return vendors.every(isPresent);
```

Comment (mvciekzvbek) [reply src/components/popup/details/vendors/vendors.jsx]: I think we could use Array.prototype.every here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every

File: src/lib/store.js [hunk 2]
```diff
@@ -72,6 +72,7 @@ export default class Store {
 		this.isFooterShowing = false;
 		this.section = SECTION_INTRO;
 		this.subsection = SECTION_PURPOSES;
+		this.hasInitialVendorsRejectionOccurs = false;
```
Comment (mvciekzvbek) [inline src/lib/store.js]: hasInitialVendorsRejectionOccur**ed** ?

File: src/components/popup/details/vendors/vendors.jsx (hunk 1 above)
Comment (osk-8) [reply src/components/popup/details/vendors/vendors.jsx]: Nice, good suggestion.

File: src/lib/store.js (hunk 2 above)
Comment (osk-8) [reply src/lib/store.js]: Right

**Review by mvciekzvbek — APPROVED** [review]

**Pull request merged**